spec/models/ability_spec.rb

```diff
@@ -2,36 +2,35 @@
 require 'cancan/matchers'
 
 describe Ability do
-  before(:each) do
-    @member = FactoryGirl.create(:member)
-    @ability = Ability.new(@member)
-  end
+
+  let(:member) { FactoryGirl.create(:member) }
+  let(:ability) { Ability.new(member) }
 
   context "notifications" do
     it 'member can view their own notifications' do
-      @notification = FactoryGirl.create(:notification, :recipient => @member)
-      @ability.should be_able_to(:read, @notification)
+      @notification = FactoryGirl.create(:notification, :recipient => member)
```

Comment:
I think this would be a good opportunity to make `@notification` a local variable - IIRC when we wrote these tests we didn't understand Ruby's scoping rules very well :-)